libs/components/core/src/lib/modules/media-query/media-query-service-interface.ts

```diff
@@ -0,0 +1,33 @@
+import { Observable, Subscription } from 'rxjs';
+
+import { SkyMediaBreakpointType } from './media-breakpoint-type';
+import { SkyMediaBreakpoints } from './media-breakpoints';
+import { SkyMediaQueryListener } from './media-query-listener';
+
+/**
+ * @internal
+ */
+export interface SkyMediaQueryServiceInterface {
```

Comment:
👀 This interface is used by `SkyMediaQueryService` and `SkyMediaQueryTestingService` to ensure all public properties/methods are the same.